Add On is now Localized and Ready for Translation

MaryOJob opened the pull request:
### Changes proposed in this Pull Request:

Prepared this Add-On for localization.

### Changelog entry

> This Add On has been localized and is translatable.

MaryOJob committed:
Made Add-On Translatable

Prepared this Add-On for localization.

MaryOJob committed:
Updated Plugin test domain function

Updated Plugin test domain function

andrewlimaza commented on pmpro-cancel-on-next-payment-date.php:
```diff
@@ -7,8 +7,17 @@
+/*
+	Load plugin textdomain.
+*/
+function pmprorh_load_conpd() {
```
This should be `pmproconpd_load_text_domain`

andrewlimaza replied:
Be sure to adjust the callback in the action too.

andrewlimaza commented on pmpro-cancel-on-next-payment-date.php:
```diff
@@ -141,7 +150,7 @@ function pmproconpd_gettext_cancel_text($translated_text, $text, $domain) {
   if(($domain == "pmpro" || $domain == "paid-memberships-pro") && $text == "Your membership has been cancelled.") {
 		global $current_user;
-		$translated_text = "Your recurring subscription has been cancelled. Your active membership will expire on " . date(get_option("date_format"), $pmpro_next_payment_timestamp) . ".";
+		$translated_text = sprintf( __( 'Your recurring subscription has been cancelled. Your active membership will expire on %s', 'pmpro-cancel-on-next-payment-date' ), date( get_option( 'date_format' ), $pmpro_next_payment_timestamp ) );
```
Please can you add a full stop at the end of the text here. (You may need to regenerate the pot files)

andrewlimaza commented on pmpro-cancel-on-next-payment-date.php:
```diff
@@ -157,7 +166,8 @@ function pmproconpd_pmpro_email_body($body, $email) {
 			if($pmpro_next_payment_timestamp - current_time( 'timestamp' ) > 0) {
-				$body .= "<p>Your access will expire on " . date(get_option("date_format"), $pmpro_next_payment_timestamp) . ".</p>";
+				$expiry_date = date( get_option( 'date_format' ), $pmpro_next_payment_timestamp );
+				$body .= '<p>' . sprintf( __( 'Your access will expire on %s', 'pmpro-cancel-on-next-payment-date' ), $expiry_date ) . '</p>';
```
This also needs a full stop added back to the string.

andrewlimaza commented on languages/gettext.sh:
```diff
@@ -0,0 +1,24 @@
+# Change every instance of pmpro-cancel-on-next-payment-date below to match your actual plugin slug
```
We can remove this line of text - Line 1 only.

MaryOJob committed:
Made corrections

- Added a . after %s strings
- Corrected the function for loading the text domain.
- Deleted line 1 in the gettext.sh file
